server/parsec/components/postgresql/user.py

```diff
@@ -487,18 +471,18 @@ async def _create_user(
         if result != "INSERT 0 1":
             assert False, f"Insertion error: {result}"
 
-        # Finally make sure there is only one non-revoked user with this human handle
+        # Finally make sure there is only one active user with this human handle
```

Comment:
Just to be sure, active means nor revoked, nor frozen  ?

Reply:
Users are either active or revoked. Only active users are taken into account when invoicing customers.

A frozen user is "temporarily" prevented from authenticating to Parsec server, but is still considered an active user. The frozen status requires an action from an admin: either revoke or unfreeze the user.